src/xalt_initialize.c

```diff
@@ -503,14 +503,24 @@ void myinit(int argc, char **argv)
   if (v == NULL)
     v = XALT_GPU_TRACKING;
   xalt_gpu_tracking = (strcmp(v,"yes") == 0);
-  
+
   do
     {
       if (xalt_gpu_tracking)
         {
           DEBUG0(stderr, "  GPU tracing\n");
 
 #ifdef USE_NVML
+          /* check whether the nvidia module is loaded */
+          char nvidiaD[] =  "/sys/module/nvidia";
+          struct stat s = {0};
```

Comment:
I assume this code compiles, but I would have thought you'd need to include `sys/stat.h`.